terminus-ui/input/src/input.component.spec.ts

```diff
@@ -106,7 +106,7 @@ describe(`TsInputComponent`, function() {
         const component = getInputInstance(fixture);
         const el = component.inputElement.nativeElement;
 
-        expect(component.readOnly).toEqual(false);
+        expect(component.readOnly).toBeFalsy();
```

Comment:
Any specific reason these expects were changed to a less strict format?

Reply:
The tests were failing. I _think_ it said expecting "false" and received "" (empty string)

Reply:
Hmm we should probably look deeper into this. We may need to update our tests to correctly use the input. `readOnly` should only be a boolean so it sounds like possibly coercion was handling an incorrect input from our test setup.